frontend/src/components/LogOutButton/LogOutButton.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ReactNode } from 'react'
+import useAuth from '../../hooks/useAuth'
+import axios from 'axios'
+
+function LogOutButton(props: { children: ReactNode }) {
+   const user = useAuth(false)
+
+   function handleLogOut() {
+      axios.post('/api/users/logout').then(() => {
+         window.location.href = '/login'
+      })
+   }
+
+   return user ? <div onClick={handleLogOut}>{props.children}</div> : null
```

Comment:
null is not a good practice, return an empty string instead
```suggestion
   return user ? <div onClick={handleLogOut}>{props.children}</div> : ''
```